### PR TITLE
Leaderboard: setup project

### PR DESCRIPTION
**Pull request by amejid**
## Milestone 1: Setup project

In this PR, the following were accomplished:-

- Made sure no linter errors
- Follow Gitflow
- Implement add functionality
- Use Webpack to compile project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse curriculum
+
+## 📝 License
```
Comment (Bondok6) [inline README.md:54]: - [OPTIONAL] I can see you are using the Microverse template, good job 👏 I suggest adding the `MIT.md` file to your project and adding the link in your `README.md`, here is the [MIT template](https://github.com/microverseinc/readme-template/blob/master/MIT.md)

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs.

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @amejid 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Your project is professional ✅
- Using Gitflow correctly ✅
- Well structured files ✅
- Professional README.md file 💯

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (amejid) [reply README.md:54]: Thank you for reviewing my project, I will make the necessary corrections as suggested.

**Comment by amejid** [comment]
Thank you for reviewing and approving my code.

**Pull request merged**